examples/computer_vision/ultralytics-bbox.py

```diff
@@ -1,25 +1,37 @@
 import os
-
-os.environ["YOLO_VERBOSE"] = "false"
-
-
 from io import BytesIO
 
+from numpy import asarray
 from PIL import Image
 from ultralytics import YOLO
 
-from datachain import C, DataChain, File
+from datachain import DataChain, File
 from datachain.model.ultralytics import YoloBBoxes
+from datachain.toolkit.ultralytics import visualize_yolo
+
+OUTPUT_DIR = "output/bbox"
 
 
 def process_bboxes(yolo: YOLO, file: File) -> YoloBBoxes:
-    results = yolo(Image.open(BytesIO(file.read())))
-    return YoloBBoxes.from_results(results)
+    # read image
+    img = Image.open(BytesIO(file.read()))
```

Comment:
can we use ImageFile here?

Reply:
Updated [here](https://github.com/iterative/datachain/pull/924/files#diff-31fc6b1b87fe919cc90d9c37eb5a2978edc44fbc3ed96e457dc51896dca240ca).

Reply:
hmm, but can we use it here? :)

Reply:
ah, I see